### ISSUE TITLE
Dependency of Avalonia.Input 0.10.12

### ISSUE THREAD

**Issue by rgustv** [Mapsui/Mapsui#1823]
**Describe the bug**
Dependency of Avalonia.Input version 0.10.12 in MapsUi.Avalonia 4.0.0-beta7 and Avalonia 11.0.0-preview5.

**To Reproduce**
1. Clean project created from the latest available Avalonia template
2. Upgrade to Avalonia 11.0.0-preview5
3. Install MapsUi.Avalonia 4.0.0-beta 7
4. Add a empty mapcontrol in the main window.

**Expected behavior**
Empty mapcontrol

**Screenshots**
<img width="473" alt="bilde" src="https://user-images.githubusercontent.com/13852434/217029764-88f2f329-490c-4796-9d86-d8713527fc18.png">

**Desktop (please complete the following information):**
 - OS: macOS Ventura, Win 11

**Comment by pauldendulk** [comment]
In your screenshot the exception comes from Snowflake.Views.Mapviews. What is that? another nuget package that uses Mapsui? Do you have the same problem if you do [the getting started](https://mapsui.com/documentation/getting-started-avalonia.html)? It can be done in [less than a minute](https://twitter.com/Mapsui/status/1506189922826391557). 

We reference Avalonia 0.10.12 and we could upgade, but it should be possible to override the Avalonia version in your project. Perhaps using binding redirect.

**Comment by rgustv** [comment]
Sorry my screenshot was from my current project and not from the demo, however the result is still the same.

I did a new test follwing the instructions on the getting started page. Here is my code:

```
using Avalonia.Controls;
namespace MapsUIAvaloniaTestProject.Views;

public partial class MainWindow : Window
{
	public MainWindow()
	{
		InitializeComponent();
		var mapControl = new Mapsui.UI.Avalonia.MapControl();
		mapControl.Map?.Layers.Add(Mapsui.Tiling.OpenStreetMap.CreateTileLayer());
		Content = mapControl;
	}
}
```

Packages installed:
<img width="459" alt="bilde" src="https://user-images.githubusercontent.com/13852434/217251778-b750da52-2795-4e2e-a69e-14f023bc2b56.png">

Still the same error message when debugging:
```
System.IO.FileNotFoundException: Could not load file or assembly 'Avalonia.Input, Version=0.10.12.0, Culture=neutral, PublicKeyToken=c8d484a7012f9a8b'. The system cannot find the file specified.

File name: 'Avalonia.Input, Version=0.10.12.0, Culture=neutral, PublicKeyToken=c8d484a7012f9a8b'
   at MapsUIAvaloniaTestProject.Views.MainWindow..ctor()
   at MapsUIAvaloniaTestProject.App.OnFrameworkInitializationCompleted() in /Users/lehu/RiderProjects/MapsUIAvaloniaTestProject/MapsUIAvaloniaTestProject/App.axaml.cs:line 20
   at Avalonia.AppBuilder.Setup() in /_/src/Avalonia.Controls/AppBuilder.cs:line 250
   at Avalonia.AppBuilder.SetupWithLifetime(IApplicationLifetime lifetime) in /_/src/Avalonia.Controls/AppBuilder.cs:line 160
   at Avalonia.ClassicDesktopStyleApplicationLifetimeExtensions.StartWithClassicDesktopLifetime(AppBuilder builder, String[] args, ShutdownMode shutdownMode) in /_/src/Avalonia.Controls/ApplicationLifetimes/ClassicDesktopStyleApplicationLifetime.cs:line 211
   at MapsUIAvaloniaTestProject.Program.Main(String[] args) in /Users/lehu/RiderProjects/MapsUIAvaloniaTestProject/MapsUIAvaloniaTestProject/Program.cs:line 13
```
Override could be a temporary solution, but the better option would perhaps be to upgrade the reference if the latest version of MapsUi are ment to support Avalonia 11.0.0 when it releases.

**Comment by pauldendulk** [comment]
I will upgrade, but in general it should be possible to run with a newer version when mapsui is built with an older version. Perhaps there was some specific incompatibility here. I just did the getting started on windows and could reproduce the problem there.

**Comment by pauldendulk** [comment]
@rgustv I upgraded to 0.10.18. I also tried to upgrade to 11.0.0-preview5 but had no immediate success. Please let me know if you have some suggestions for fixes: https://github.com/Mapsui/Mapsui/pull/1829.

**Comment by rgustv** [comment]
@pauldendulk I will investigate it further next week to see if there is some way around this. It seems that the 11-version of Avalon’s has some breaking changes compared with the previous releases.

**Comment by pauldendulk** [comment]
The new release (beta.8) uses 0.10.18.

**Comment by IldarKhayrutdinov** [comment]
Avalonia released [0.10.19](https://github.com/AvaloniaUI/Avalonia/releases/tag/0.10.19)

**Comment by pauldendulk** [comment]
In Mapsui 4.0.0-rc.3, which was released yesterday Mapsui.Avalonia was upgraded to the latest Avalonia rc and a new package Mapsui.Avalonia.v0 was introduced to keep supporting the V0.10 line.
